public/search.js

```diff
@@ -97,6 +103,19 @@ function displayAxis(times)
   }
 }
 
+var getDocColl = function(uri) {
+  $.ajax({
+    url: '/v1/documents?uri='+uri+'&category=collections&format=json',
+    success: function(data, textStatus) {
+      console.log('got collections: ' + data);
+    },
+    error: function(jqXHR, textStatus, errorThrown) {
+      console.log('problem');
+    },
+    async: false,
+  });
+}
+
```

Comment:
This function is defined but not used in search.js. also it is already defined in another .js file.